src/test/java/com/javachallengers/simpson/controller/SimpsonCharacterControllerIntegrationTest.java

```diff
@@ -0,0 +1,112 @@
+package com.javachallengers.simpson.controller;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.apache.http.HttpHeaders;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+
+import com.javachallengers.simpson.application.SimpsonApplication;
+import com.javachallengers.simpson.model.dto.SimpsonCharacterRequestDTO;
+
+import io.restassured.RestAssured;
+
+/**
+ * Integration tests of {@link SimpsonCharacterController}
+ */
+@Tag("IntegrationTest")
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, classes = SimpsonApplication.class)
+@TestMethodOrder(OrderAnnotation.class)
+class SimpsonCharacterControllerIntegrationTest {
+	@LocalServerPort
+	private int port;
+
+	@BeforeEach
```

Comment:
Amazing! It's great to see that you added integration-tests here!! Right on!

Reply:
Yes. I just need to implement in-memory mongodb yet. I used local mongo to run the integration tests.

Reply:
That would be awesome!!

Reply:
Added in-memory mongoDB for integration tests.